editor/editor_help.cpp

```diff
@@ -177,6 +177,20 @@ void EditorHelp::_class_desc_resized() {
 	class_desc->add_theme_style_override("normal", class_desc_stylebox);
 }
 
+String EditorHelp::_convert_case(const String &p_name) {
+	switch ((int)EDITOR_GET("text_editor/help/class_reference_language")) {
+		case 0: // GDScript
+			return p_name;
+		case 1: {
+			// C#
+			bool first_is_underscore = p_name[0] == '_';
+			String result = p_name.capitalize();
+			return (first_is_underscore ? "_" : "") + result.replace(" ", "");
```

Comment:
Maybe we should make a method on String to do this? I think it would be more efficient, and we could make a method that doesn't involve string splitting and re-concatenation - we can simply capitalize every letter after `_` (and at the beginning) and then afterwards do a find/replace `"_"` with `""` (except keep one at the beginning).

Reply:
I'd try to benchmark that code to see if it's an actual bottleneck first. Adding more code to `ustring.cpp` should be done cautiously.

Reply:
Will do, but this code is not executed in a performance critical part of the engine. I can test wheter or not a iteration above the whole string is faster